src/lib/utils/thorchain/utils/getMimir.ts

```diff
@@ -0,0 +1,8 @@
+import type { ThorchainMimir } from '../types'
+
+export const getMimir = <T extends keyof ThorchainMimir>(
```

Comment:
Credits where it's due: https://gitlab.com/thorchain/thornode/-/blob/c979858d83106d8959c1140365231a1a6492b837/x/thorchain/keeper/v1/keeper_halt.go#L77

Could just access it directly, but nice QoL util!

Reply:
Meh, actually useless 007840d6f5